CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+
+We particularly encourage contributors from underrepresented backgrounds to engage and join our community. Please note that all Nextstrain projects follow and enforce the [Contributor Covenant Code of Conduct](https://www.contributor-covenant.org/).
+
+# Contributing code
+Nextstrain comprises [multiple interconnected software components](https://github.com/nextstrain/). Below is a list of Nextstrain’s primary components with links to their active issues.
+
+- [nextstrain.org](https://github.com/nextstrain/nextstrain.org/issues): static content and documentation for the nextstrain.org website
+- [auspice](https://github.com/nextstrain/auspice/issues): front-end visualization application 
+- [augur](https://github.com/nextstrain/augur/issues): back-end bioinformatics tools 
+- [cli](https://github.com/nextstrain/cli/issues): unified command-line interface for analyzing and visualizing data 
+- [ncov](https://github.com/nextstrain/ncov/issues): hCoV-19 data analysis pipeline 
+
+## Quick start
+
+1. Find an open issue that matches your skills and interests [on a Nextstrain project board](https://github.com/orgs/nextstrain/projects) or [among all issues for Nextstrain components listed above](https://github.com/search?q=org%3Anextstrain&state=open&type=Issues).
+
+2. **Comment on the issue**, to let us know you’d like us to assign that issue to you. Include a brief description of how you plan to address the issue and, optionally, any prototypes or mockups of your planned implementation. We will respond to your comment to let you know you’ve been assigned or to clarify aspects of your proposed solution.
+
+3. **Submit your solution** to the selected issue as a pull request (PR). We will review your contribution, offer constructive feedback, and merge your solution with the repository when all outstanding review comments have been resolved.
+
+_If you're new to Github, check out [this awesome list of resources](https://github.com/freeCodeCamp/how-to-contribute-to-open-source#using-version-control) to help get you started!_
+
+## Development details
+
+Please see `DEV_DOCS.md` in the relevant repository for guidance on how to set up your dev environment, etc. 
+
+We use Python for back-end components and loosely follow [the PEP 8 standard](https://www.python.org/dev/peps/pep-0008/). We use Javascript for front-end components and use [eslint](https://eslint.org/) to enforce [a custom set of linting rules](https://github.com/nextstrain/auspice/blob/master/.eslintrc).
+
+# Improve Nextstrain's software without code  
+
+## Report a bug
+
+If you think you’ve found a bug in a component of Nextstrain, [first search for an existing bug report across all issues in the organization](https://github.com/search?q=org%3Anextstrain&state=open&type=Issues).
+
+![](https://i.imgur.com/fAJEFWH.png)
```

Comment:
This image should be hosted on GitHub inside this repo. I'll work on adding this today or tomorrow.